client/packages/system/src/Documents/DocumentsTable.tsx

```diff
@@ -20,58 +18,63 @@ interface DocumentsProps {
   recordId: string;
   tableName: string;
   documents: SyncFileReferenceFragment[];
-  noDataElement?: JSX.Element;
-  openUploadModal?: () => void;
+  isFetching?: boolean;
   invalidateQueries?: () => void;
+  openUploadModal?: () => void;
 }
 
 export const DocumentsTable = ({
   recordId,
   tableName,
   documents,
-  noDataElement,
-  openUploadModal,
+  isFetching,
   invalidateQueries,
+  openUploadModal,
 }: DocumentsProps): ReactElement => {
   const t = useTranslation();
 
-  const columns = useColumns<SyncFileReferenceFragment>([
-    GenericColumnKey.Selection,
-    {
-      key: 'fileName',
-      label: 'label.filename',
-      accessor: ({ rowData }) => rowData.fileName,
-    },
-    {
-      key: 'createdDatetime',
-      label: 'label.created-datetime',
-      accessor: ({ rowData }) => rowData.createdDatetime,
-      format: ColumnFormat.Date,
-    },
-  ]);
+  const columns = useMemo(
+    (): ColumnDef<SyncFileReferenceFragment>[] => [
+      {
+        header: t('label.filename'),
+        accessorKey: 'fileName',
+        enableSorting: true,
+      },
+      {
+        header: t('label.created-datetime'),
+        accessorKey: 'createdDatetime',
+        columnType: ColumnType.Date,
+        enableSorting: true,
+      },
+    ],
+    []
+  );
```

Comment:
This is fine as far as replacing what's there, but we should probably change this to not be a table at some point, as we discussed. You wanna chuck a "TO-DO" in here, or make a new issue?